plugins/PrivacyManager/Model/DataSubjects.php

```diff
@@ -135,9 +135,13 @@ private function invalidateArchives($datesToInvalidateByIdSite)
         $invalidator = StaticContainer::get('Piwik\Archive\ArchiveInvalidator');
 
         foreach ($datesToInvalidateByIdSite as $idSite => $visitDates) {
+            $idSites = [$idSite];
+            Piwik::postEvent('Archiving.getIdSitesToMarkArchivesAsInvalidated', array(&$idSites, $visitDates, null, null, null, $isPrivacyDeleteData = true));
```

Comment:
Probably not a big deal but this is introducing an unused variable `$isPrivacyDeleteData` that was probably just copied over from the event definition whereas just the `true` value would suffice here.

Reply:
We have that in many places of the code, to make it easier to see which parameter that actually is.
Normally I would try to avoid such stuff, as in most cases the IDE would already show, which parameter that is. But in case of events that won't be shown. So when reading the code it would be helpful in that case, as you won't need to look up where the event is fired.

Reply:
Should/Would we do it consistently for all the params? Or are `null`s special?
Once we drop PHP 7.x support we might be able to use named arguments, but that is probably a long way out. Would be nice though!

Reply:
I would see null as special case, as long as it is the default value for the event. But guess it wouldn't hurt to have it nevertheless.

Reply:
All good, thanks for the insights. We can leave it as is for now.